code/modules/mapfluff/ruins/spaceruin_code/hilbertshotel.dm

```diff
@@ -504,10 +504,10 @@ GLOBAL_VAR_INIT(hhMysteryRoomNumber, rand(1, 999999))
 			to_chat(user, "No vacated rooms.")
 		return .
 
-/obj/effect/landmark/lift_id/hilbert/research
+/obj/effect/landmark/lift_id/hilbert
 	specific_lift_id = HILBERT_TRAM
```

Comment:
> > Corrects the specific_tram_id helper for the Hilbert's Research to use the hilbert tram ID.
> 
> It seems you've forgotten to actually do it. The 'Files changed' section only shows the changes to the paths.

This line is, effectively, the fix.
before, the map was using the `.../hilbert` lift id landmark. Since specific_lift_id was only set on `.../hilbert/research`, the hilbert tram wasn't actually getting set up correctly.